deploy/files.sql

```diff
@@ -34,13 +39,14 @@ CREATE TABLE mappa.files (
   created_at  TIMESTAMPTZ,
   user_id     UUID,
   name        TEXT,
+  org         TEXT,
```

Comment:
could you make this JSONB so that we can do structured queries against it later?